packages/qwik/src/utils/getFormDataValues.ts

```diff
@@ -80,17 +80,20 @@ export function getFormDataValues<TFieldValues extends FieldValues>(
           return (object[key] =
             object[key] || (isNaN(+keys[index + 1]) ? {} : []));
         }
-
-        // Otherwise, if it is not an emty file, add value
+        // Otherwise, if it is not an empty file, add value
         if (
           !files.includes(template) ||
           (value && (typeof value === 'string' || value.size))
         ) {
           if (arrays.includes(template)) {
             object[key] = object[key] || [];
             object[key].push(getValue());
-          } else {
-            object[key] = getValue();
+          } else if (files.includes(template)) {
```

Comment:
I think we should replace `getFormDataValues` with my npm package `decode-formdata`. I implemented it based on this util but fixed a lot of bugs. `decode-formdata` has 100% test coverage.

Reply:
Go ahead, I just added that one case for files